packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -37,8 +38,14 @@ const isInvalidFileName = (fileName: string): boolean => {
     return !!fileName.match(invalidCharsRegex);
 };
 
+// This map represents mapping from container package name to file extension.
+// For the packages that are not in this list, we will use .fluid extension.
+const fileExtensionMap = new Map([
+    [encodeURIComponent("@ms/onenote-meetings"), ".note"],
```

Comment:
I talked w/ Cosmin and he feels that maybe this shouldn't be app specific at all... the alternate proposal for this string is something like @ms/meetings-container. What do you think? I'll stop changing the string once you check this in :P

Reply:
But shouldn't file extension be app specific? Like all meetings need not be done in onenote?

Reply:
Yeah it does feel like this is currently very app specific because of how it's being used here to literally define ".note". I think the thinking is that maybe one day this could change to opening in another app by default... but I don't know what that means for all files already created with .note extensions. We can leave this way for now, since we are doing directly with ".note" as the extension.